001-quickstart-passwordless/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// ------------------------------------------------------------
+
+// <using_directives> 
+using Microsoft.Azure.Cosmos;
+using Azure.Identity;
+// </using_directives>
+
+// <client_credentials> 
+// New instance of CosmosClient class
+using CosmosClient client = new(
+    "https://<your-cosmosdb-account-name>.documents.azure.com:443/",
+    new DefaultAzureCredential()
+);
+// </client_credentials>
+
+// <new_database> 
+// Database reference with creation if it does not already exist
+Database database = client.GetDatabase("<your-database-name>");
+
+Console.WriteLine($"New database:\t{database.Id}");
+// </new_database>
+
+// <new_container> 
+// Container reference with creation if it does not alredy exist
+Container container = database.GetContainer("<your-container-name>");
```

Comment:
```suggestion
Container container = database.GetContainer(
    id: "products"
);
```

We were also prescriptive about the name here.